### PR TITLE
Working changes related to Creation of graph and new features for navigation

### PR DESCRIPTION
**Pull request by Adij1110**
Working changes related to Creation of graph and new features for navigation
Navigation- o test navigation via different links
NewGraph- Creation of new graph and verification of other features related to it.
NewGraphValidation- To test validation for creation of new graph.

### CODE REVIEW

File: cypress/support/step_definitions/pages/landing.ts
Comment (abhishekkundu25) [inline cypress/support/step_definitions/pages/landing.ts:1]: format file with prettier

File: cypress/support/e2e.ts [hunk 1]
```diff
@@ -1,44 +1,98 @@
...
+        deleteItem(0);
+      });
+    } else {
+      cy.log("No graph to delete");
+    }
+  });
+}
+
+// afterEach(() => {
+//   deleteAllGraphs();
```
Comment (abhishekkundu25) [inline cypress/support/e2e.ts:96]: remove commented code

**Review by bhaveshgadhegit — APPROVED** [review]

**Review by bhaveshgadhegit — APPROVED** [review]
Functional review done

**Review by abhishekkundu25 — APPROVED** [review]

**Comment by Abdulazeem84** [comment]
Functional Review Done

**Pull request merged**